assets/css/materialize.css

```diff
@@ -2897,6 +2898,40 @@ ul.staggered-list li {
   list-style-type: none;
 
 }
+.comparison {
+          float:left; display: inline; 
```

Comment:
Good work, but no need for indentation here.

Reply:
Okay

Reply:
@subhamkrai looking good! Your changes have been deployed to the site. Looks nice.

Reply:
@leecalcote thanks